Packs/GitHub/Integrations/GitHub/GitHub_test.py

```diff
@@ -74,6 +74,33 @@ def mock_http_request(method, url_suffix, params=None, data=None, headers=None,
 ]
 
 
+RETURN_ERROR_TARGET = 'GitHub.return_error'
+
+
+@pytest.mark.parametrize('params, expected_result', [
+    ({'credentials': {'password': '1234'}, '_auth_id': '_auth_id'}, "Insert api token or private key"),
```

Comment:
Shouldn't this succeed? There is an API token in the request

Reply:
It was a mistake adding the _auth_id token.
however, this will fail because both Token and private key will come out empty